core/modules/redirect/redirect.admin.inc

```diff
@@ -578,7 +578,10 @@ function _redirect_extract_url_options(&$element, &$form_state) {
  */
 function redirect_edit_form_validate($form, &$form_state) {
   form_state_values_clean($form_state);
-  $redirect = new Redirect($form_state['values']);
+  $values = $form_state['values'];
+  unset($values['override']);
+  unset($values['delete']);
```

Comment:
Can we please add an inline comment here, explaining why we are unsetting these?

Reply:
Makes sense, noted. :+1: